docs/content/blog/persist-sensitive-data-safely.md

```diff
@@ -0,0 +1,85 @@
+Now Porter requires users to configure a secret store to hold any data that has been flaged as sensitive in a bundle. 
+
+Let's walk through how to utilize this new feature by updating your Porter configuration file and picking a secret plugin that works best for your usecase.
+
+If you haven't installed the latest Porter v1 release, please follow the instruction [here](https://release-v1.porter.sh/install/#prerelease).
+
+After the installation, let's try to install a bundle that works with sensitive data with just the default configurations in Porter.
+
+```
+porter install --reference ghcr.io/getporter/examples/sensitive-data --param password=123a123
+```
+
+You should see below error message in the output from the above command:
+```
+failed to save sensitive param to secrete store: rpc error: code = Unknown desc = The default secrets plugin, secrets.porter.host, does not support persisting secrets: not implemented
+```
+
+In this example bundle, there's a sensitive parameter named as `password` and a sensitive output called `name`.
+
+By default, Porter does not provide a built-in solution to persist sensitive data. Any bundle that references or produces sensitive data will fail to execute. 
+
+Now let's configure Porter to persist sensitive data using plain text file through the [filesystem](https://release-v1.porter.sh/plugins/filesystem/) plugin.
+
+```yaml
+default-secrets-plugin: "filesystem"
+```
+
+The [filesystem](https://release-v1.porter.sh/plugins/filesystem/) plugin resolves and stores sensitive bundle parameters and outpus as plaintext files in your PORTER_HOME directory.
+Note: the filesystem plugin is only intended for testing and local development usage. It's not intended to be used in production. Please see at the end of this post on how to pick a plugin that is suitable for production usecase.
+
+Now you have a secret store set up, we can try to install the example bundle again.
+
+```
+porter install --reference ghcr.io/getporter/examples/sensitive-data --param password=123a123
+```
+
+Once the installation process finishes, you should see outputs like below:
+
+```
+executing install action from sensitive-data (installation: /sensitive-data)
+Install Hello World
+Hello, installing example-bundle with password: *******
+execution completed successfully!
+```
+
+If you inspect your Porter storage database, in this case, it's a [mongodb instance](https://release-v1.porter.sh/plugins/mongodb-docker/), you should see the value of the `password` parameter is a reference to a [secret](https://release-v1.porter.sh/parameters/#user-specified-values).
```

Comment:
The implementation details here are a bit too detailed for a blog post. Let's just leave it at something like "Porter stores a reference to the sensitive data that was saved in the configured secret store."